### PR TITLE
[doc] Use Rainbond deployment DolphinScheduler Cluster

### PR DESCRIPTION
**Pull request by zzzhangqi**
Signed-off-by: Qi Zhang <smallqi1@163.com>

## Purpose of the pull request

This pull request add new installation,use Rainbond deploy DolphinScheduler cluster.

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

File: docs/docs/en/guide/installation/rainbond.md
Comment (zhongjiajie) [inline docs/docs/en/guide/installation/rainbond.md]: maybe we should directly tell the user search the keyword `dolphinscheduler` instead of part of it
```suggestion
* Docking and accessing the built-in open source app store,Search the keyword `dolphinScheduler` to find App DolphinScheduler.
```

File: docs/docs/en/guide/installation/rainbond.md
Comment (zhongjiajie) [inline docs/docs/en/guide/installation/rainbond.md]: can you add this img to our `img` directory? we need to hold those images instead of using extracting link

File: docs/docs/en/guide/installation/rainbond.md
Comment (zhongjiajie) [inline docs/docs/en/guide/installation/rainbond.md]:
```suggestion

```

File: docs/docs/en/guide/installation/rainbond.md
Comment (zhongjiajie) [inline docs/docs/en/guide/installation/rainbond.md]:
```suggestion

```

File: docs/docs/zh/guide/installation/rainbond.md
Comment (zhongjiajie) [inline docs/docs/zh/guide/installation/rainbond.md]:
```suggestion

```

File: docs/docs/zh/guide/installation/rainbond.md
Comment (zhongjiajie) [inline docs/docs/zh/guide/installation/rainbond.md]:
```suggestion

```

File: docs/docs/en/guide/installation/rainbond.md
Comment (zhongjiajie) [inline docs/docs/en/guide/installation/rainbond.md]: I found out in rainbond, we do not have English version of it interface. but in English we still need to translate it to English

**Review by zhongjiajie — COMMENTED** [review]
One more thing, maybe we should put this doc in new section named `integrated` better than `Installation`

File: docs/docs/en/guide/installation/rainbond.md
Comment (zzzhangqi) [reply docs/docs/en/guide/installation/rainbond.md]: This is indeed a deficiency of the product,no english ui

**Comment by zzzhangqi** [comment]
> One more thing, maybe we should put this doc in new section named `integrated` better than `Installation`

`docs/docs/zh/guide/installation/integrated/rainbond.md`
Put the doc in this directory?

File: docs/docs/en/guide/installation/rainbond.md
Comment (zzzhangqi) [reply docs/docs/en/guide/installation/rainbond.md]: No problem

File: docs/docs/en/guide/installation/rainbond.md
Comment (zzzhangqi) [reply docs/docs/en/guide/installation/rainbond.md]: `通用数据初始化插件` In the doc, I changed it from Chinese to English.

**Comment by zzzhangqi** [comment]
@zhongjiajie proposal has been dealt with.please review

**Comment by SbloodyS** [comment]
I had approved to run CI.

**Comment by zzzhangqi** [comment]
@SbloodyS I don't quite understand why ci didn't pass.Can you give me some guidance?

**Comment by SbloodyS** [comment]
> @SbloodyS I don't quite understand why ci didn't pass.Can you give me some guidance?

Sorry for late reply. You can find the detail of errors in `https://github.com/apache/dolphinscheduler/runs/7368120405?check_suite_focus=true`. And you can find this url in the `Dead Link's Detail`.

**Comment by zzzhangqi** [comment]
@SbloodyS All checks have passed.Please review

**Comment by SbloodyS** [comment]
@zhongjiajie @EricGao888 PTAL.

File: docs/docs/en/guide/installation/rainbond.md
Comment (EricGao888) [inline docs/docs/en/guide/installation/rainbond.md]:
```suggestion
This section describes the one-click deployment of high availability DolphinScheduler clusters through the [Rainbond](https://www.rainbond.com/) cloud native application management platform. This method is suitable for users who don't know much about complex technologies such as `Kubernetes`, lowering the threshold for deploying DolphinScheduler in `Kubernetes` mode.
```

File: docs/docs/en/guide/installation/rainbond.md
Comment (EricGao888) [inline docs/docs/en/guide/installation/rainbond.md]:
```suggestion
* Available Rainbond cloud native application management platform is a prerequisite，please refer to the official `Rainbond` documentation [Rainbond Quick install](https://www.rainbond.com/docs/quick-start/quick-install)
```

File: docs/docs/en/guide/installation/rainbond.md
Comment (EricGao888) [inline docs/docs/en/guide/installation/rainbond.md]:
```suggestion
* Logging in and accessing the built-in open source app store, search the keyword `dolphinscheduler` to find the DolphinScheduler App.
```

Comment (EricGao888) [inline docs/docs/en/guide/installation/rainbond.md]: May I ask what `docking and accessing the built-in open source app store` means here? Is the app store in `Rainbond`?

File: docs/docs/en/guide/installation/rainbond.md
Comment (EricGao888) [inline docs/docs/en/guide/installation/rainbond.md]:
```suggestion
* Click `install` on the right side of DolphinScheduler to go to the installation page. Fill in the corresponding information and click `OK` to start the installation. You will get automatically redirected to the application view.
```

File: docs/docs/en/guide/installation/rainbond.md
Comment (EricGao888) [inline docs/docs/en/guide/installation/rainbond.md]:
```suggestion
* Wait a few minutes, the installation completes and `DolphinScheduler` will be running.
```

File: docs/docs/en/guide/installation/rainbond.md
Comment (EricGao888) [inline docs/docs/en/guide/installation/rainbond.md]:
```suggestion
* Access DolphinScheduler-API components and the default user credentials are `admin` / `dolphinscheduler123`.
```

File: docs/docs/en/guide/installation/rainbond.md
Comment (EricGao888) [inline docs/docs/en/guide/installation/rainbond.md]:
```suggestion
DolphinScheduler API、Master、Worker all support scaling multiple instances, ensuring the high availability of the entire service.
```

File: docs/docs/en/guide/installation/rainbond.md
Comment (EricGao888) [inline docs/docs/en/guide/installation/rainbond.md]:
```suggestion
Take `worker` as an example: enter the `component -> Telescopic` page, and set the number of instances.
```

File: docs/docs/en/guide/installation/rainbond.md
Comment (EricGao888) [inline docs/docs/en/guide/installation/rainbond.md]:
```suggestion
To verify `worker` node, enter `DolphinScheduler UI -> Monitoring -> Worker` page to view detailed node information.
```

File: docs/docs/en/guide/installation/rainbond.md
Comment (EricGao888) [inline docs/docs/en/guide/installation/rainbond.md]:
```suggestion
API and Worker Services share the configuration file `/opt/dolphinscheduler/conf/common.properties`. To modify the configurations, you only need to modify that of the API service.
```

File: docs/docs/en/guide/installation/rainbond.md
Comment (EricGao888) [inline docs/docs/en/guide/installation/rainbond.md]:
```suggestion
Worker service is installed with default `Python3`，you can add environment variables `PYTHON_HOME=/usr/bin/python3`
```

File: docs/docs/en/guide/installation/rainbond.md
Comment (EricGao888) [inline docs/docs/en/guide/installation/rainbond.md]:
```suggestion
Take `DataX` as an example:
```

File: docs/docs/en/guide/installation/rainbond.md
Comment (EricGao888) [inline docs/docs/en/guide/installation/rainbond.md]:
```suggestion
3. Update component, the plug-in `Datax` will be downloaded automatically and decompress to `/opt/soft`
```

File: docs/docs/en/guide/installation/rainbond.md
Comment (EricGao888) [inline docs/docs/en/guide/installation/rainbond.md]:
```suggestion
2. Open plugin.enter Worker component -> plugin -> open `initialization plugin` and modify the configuration.
```

**Comment by EricGao888** [comment]
> @zhongjiajie @EricGao888 PTAL.

Hi @zzzhangqi, I just added some comments, thx

File: docs/docs/en/guide/installation/rainbond.md
Comment (zzzhangqi) [reply docs/docs/en/guide/installation/rainbond.md]: > May I ask what `docking and accessing the built-in open source app store` means here? Is the app store in `Rainbond`?

Rainbond app store is saas service,need to register users,so need authorization in builtin open source app store

Comment (EricGao888) [reply docs/docs/en/guide/installation/rainbond.md]: > > May I ask what `docking and accessing the built-in open source app store` means here? Is the app store in `Rainbond`?
> 
> Rainbond app store is saas service,need to register users,so need authorization in builtin open source app store

I got it. Thx for the explanation : )

**Comment by zzzhangqi** [comment]
Hi @EricGao888 ,I have revised your suggest.please approve run ci. 🤗

**Comment by EricGao888** [comment]
> Hi @EricGao888 ,I have revised your suggest.please approve run ci. 🤗

@zzzhangqi Hi, thx for the update. CI already seems running. GitHub CI has been extremely slow recently, all we could do is wait. 🤣

**Comment by zzzhangqi** [comment]
@EricGao888 Hi, All checks have passed.Please again  review.😜

**Review by EricGao888 — DISMISSED** [review]
LGTM

**Comment by zzzhangqi** [comment]
> LGTM

Thanks,This pr auto merge?

**Comment by EricGao888** [comment]
> > LGTM
> 
> Thanks,This pr auto merge?

@zzzhangqi We need two approvals here to get it merged. Maybe you could ping another committer if you're hurried to get it in : )

**Comment by zzzhangqi** [comment]
> > > LGTM
> > 
> > 
> > Thanks,This pr auto merge?
> 
> @zzzhangqi We need two approvals here to get it merged. Maybe you could ping another committer if you're hurried to get it in : )

Thanks 😘

**Comment by zzzhangqi** [comment]
@zhongjiajie Hi, Please review, Thanks.

**Review by zhongjiajie — COMMENTED** [review]
Can we remove img `monitor-dolphinscheduler.png` and `homepage-dolphinscheduler.png` and it related content? it is duplicate to exists https://dolphinscheduler.apache.org/en-us/docs/dev/user_doc/guide/homepage.html and https://dolphinscheduler.apache.org/en-us/docs/dev/user_doc/guide/monitor.html

maybe you add a hyper link to above link, or directly use pure markdown content without img

**Comment by zhongjiajie** [comment]
And BTW, I think it belong to section `integration` instead of `deployment`, maybe we should add a new sidebar named `integration` to hold this or other content like that, WDTY @EricGao888 @zzzhangqi

**Comment by zhongjiajie** [comment]
> > > LGTM
> > 
> > 
> > Thanks,This pr auto merge?
> 
> @zzzhangqi We need two approvals here to get it merged. Maybe you could ping another committer if you're hurried to get it in : )

FYI, @EricGao888 currently we only need one approval to merge PR

**Comment by EricGao888** [comment]
> And BTW, I think it belong to section `integration` instead of `deployment`, maybe we should add a new sidebar named `integration` to hold this or other content like that, WDTY @EricGao888 @zzzhangqi

Sounds good to me.

**Comment by zzzhangqi** [comment]
> https://dolphinscheduler.apache.org/en-us/docs/dev/user_doc/guide/homepage.html

No problem

**Comment by zzzhangqi** [comment]
> Can we remove img `monitor-dolphinscheduler.png` and `homepage-dolphinscheduler.png` and it related content? it is duplicate to exists https://dolphinscheduler.apache.org/en-us/docs/dev/user_doc/guide/homepage.html and https://dolphinscheduler.apache.org/en-us/docs/dev/user_doc/guide/monitor.html
> 
> maybe you add a hyper link to above link, or directly use pure markdown content without img

No problem

**Comment by zzzhangqi** [comment]
> And BTW, I think it belong to section `integration` instead of `deployment`, maybe we should add a new sidebar named `integration` to hold this or other content like that, WDTY @EricGao888 @zzzhangqi

move to `dev/user_doc/guide/installation/integration` ? @EricGao888 @zhongjiajie  Is that okay?

or `dev/user_doc/integration/rainbond.html` first level menu?

**Comment by zzzhangqi** [comment]
Hi @zhongjiajie , May I ask how to deal with this PR next? What should I do?

**Comment by EricGao888** [comment]
> > And BTW, I think it belong to section `integration` instead of `deployment`, maybe we should add a new sidebar named `integration` to hold this or other content like that, WDTY @EricGao888 @zzzhangqi
> 
> move to `dev/user_doc/guide/installation/integration` ? @EricGao888 @zhongjiajie Is that okay?
> 
> or `dev/user_doc/integration/rainbond.html` first level menu?

I prefer `dev/user_doc/integration/rainbond.html`. WDYT @SbloodyS @zhongjiajie

**Comment by zzzhangqi** [comment]
> > > And BTW, I think it belong to section `integration` instead of `deployment`, maybe we should add a new sidebar named `integration` to hold this or other content like that, WDTY @EricGao888 @zzzhangqi
> > 
> > 
> > move to `dev/user_doc/guide/installation/integration` ? @EricGao888 @zhongjiajie Is that okay?
> > or `dev/user_doc/integration/rainbond.html` first level menu?
> 
> I prefer `dev/user_doc/integration/rainbond.html`. WDYT @SbloodyS @zhongjiajie

I agree, I will modify according to `dev/user_doc/integration/rainbond.html`? @SbloodyS @zhongjiajie @EricGao888

**Comment by zzzhangqi** [comment]
How to deal with this PR next? @EricGao888 @SbloodyS @zhongjiajie

File: docs/docs/en/guide/installation/rainbond.md
Comment (EricGao888) [inline docs/docs/en/guide/installation/rainbond.md]: Please move `docs/docs/en/guide/installation/rainbond.md` to `docs/docs/en/guide/integration/rainbond.md`. The same with the Chinese docs. Other LGTM.

Comment (zzzhangqi) [reply docs/docs/en/guide/installation/rainbond.md]: ok, no problem

**Comment by zzzhangqi** [comment]
I have modified @EricGao888 , Please review

**Review by EricGao888 — APPROVED** [review]
LGTM as long as CI passes. PTAL when available @SbloodyS @zhongjiajie Thanks.

**Comment by EricGao888** [comment]
By adding different integrations with DS, it helps enrich the ecosystem. If there are no more comments on this PR, I will proceed to merge it. @SbloodyS @zhongjiajie

**Review by SbloodyS — APPROVED** [review]
+1

**Comment by EricGao888** [comment]
@zzzhangqi Thanks for contributing this to DS! Good job!

**Comment by ruanwenjun** [comment]
@SbloodyS Please optimize the commits log when merging PR. 

image

**Comment by SbloodyS** [comment]
> @SbloodyS Please optimize the commits log when merging PR.
> 
> image

I'll take care of it next time. Thanks.

**Pull request merged**
